### ISSUE TITLE
Continuous action training, is there a demo for saving and reading state_dict?

### ISSUE THREAD

**Issue by jaried** [aai-institute/tianshou#553]
-  I have marked all applicable categories:
    +  exception-raising bug
    +  RL algorithm bug
    +  documentation request (i.e. "X is missing from the documentation.")
    +  new feature request
-  I have visited the [source website](https://github.com/thu-ml/tianshou/)
-  I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
-  I have mentioned version numbers, operating system and environment, where applicable:
  ```python
import tianshou, torch, numpy, sys
print(tianshou.__version__, torch.__version__, numpy.__version__, sys.version, sys.platform)
0.4.6.post1 1.10.2+cu113 1.20.3 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)] win32
  ```
The code I use is:
`test\continuous\test_sac_with_il.py`

The `env` used is written by myself.

I have the following questions:

1. Due to the long training time, the training may be terminated early at any time. Is there a related demo for saving state_dict and reading state_dict?

2. I know how to use tensorboard in a normal environment. During the training process of Tianshou, how to use tensorboard to check the loss, reward, etc. during the training process?

**Comment by Trinkle23897** [comment]
> Due to the long training time, the training may be terminated early at any time. Is there a related demo for saving state_dict and reading state_dict?

https://tianshou.readthedocs.io/en/master/tutorials/cheatsheet.html#resume-training-process

> I know how to use tensorboard in a normal environment. During the training process of Tianshou, how to use tensorboard to check the loss, reward, etc. during the training process?

https://tianshou.readthedocs.io/en/master/tutorials/logger.html
Same as your previous usage.
